lib/registries/addon/overview/index/controller.ts

```diff
@@ -13,11 +13,20 @@ export default class Overview extends Controller {
     registration?: Registration;
 
     @computed('registration.{root,isRoot}')
-    get root() {
+    get rootTitle() {
         if (!this.registration || this.registration.isRoot) {
             return undefined;
         }
+        const rootTitle = this.registration.root.get('title');
```

Comment:
This is still making an additional AJAX call to get the asynchronous `root` relationship. Also, since asynchronous relationship returns promises, `rootTitle` here would probably be either `null` or `undefined` at the time of evaluation because the promise `this.registration.root` has yet to resolve.

Reply:
Taking a look.

Reply:
This has been reverted in favor of changes in the model.